### PR TITLE
multi: Cleanup the unsupported dcrwallet commands.

### PR DESCRIPTION
**Pull request by ggoranov**
This cleans up both dcrjson and rpcclient packages from methods
added from btcctl fork but not implemented by dcrwallet.

dcrjson methods removed:

- ImportAddressCmd
- ImportPubKeyCmd
- SendToSSGenCmd
- SendToSSRtxCmd
- SendToSStxCmd
- SetBalanceToMaintainCmd
- SetTicketMaxPriceCmd

rpcclient methods removed:

- ImportAddress, ImportAddressAsync
- ImportAddressRescan, ImportAddressRescanAsync
- ImportPubKey, ImportPubKeyAsync
- ImportPubKeyRescan, ImportPubKeyRescanAsync
- SendToSStx, SendToSStxAsync
- SendToSStxMinConf, SendToSStxMinConfAsync
- SendToSSGen, SendToSSGenAsync
- SendToSStxComment, SendToSStxCommentAsync
- SendToSSGenComment, SendToSSGenCommentAsync
- SendToSSRtxAsync, SendToSSRtxCommentAsync

As a result dcrctl does not list anymore these commands:

- importaddress
- importpubkey
- sendtossgen
- sendtossrtx
- sendtosstx
- setbalancetomaintain
- setticketmaxprice

Related to [1472](https://github.com/decred/dcrd/issues/1472)

### CODE REVIEW

File: blockchain/go.mod [hunk 1]
```diff
@@ -8,7 +8,7 @@ require (
 	github.com/decred/dcrd/dcrec v0.0.0-20180801202239-0761de129164
 	github.com/decred/dcrd/dcrec/edwards v0.0.0-20180721031028-5369a485acf6
 	github.com/decred/dcrd/dcrec/secp256k1 v1.0.0
-	github.com/decred/dcrd/dcrjson v1.0.0
+	github.com/decred/dcrd/dcrjson v1.0.1
```
[the remark this answers is not in the document]
Comment (ggoranov) [reply blockchain/go.mod]: Make sense! I'll remove the mod file changes.

**Comment by davecgh** [comment]
@ggoranov Please comment once you've updated this PR so a notification e-mail is sent.  Thanks.

**Comment by ggoranov** [comment]
Ready for review.

**Comment by jrick** [comment]
This is not correct.  Please read the [Modules](https://github.com/golang/go/wiki/Modules) entry on the Go wiki, specifically the section on Semantic Import Versioning.

**Comment by davecgh** [comment]
@jrick I asked him to roll back the version changes that were made, because they were incorrect.  Since this requires a major version bump, and none have been done in the repo yet, I'm going to create a separate PR which bumps the major versions of both `dcrjson` and `rpcclient` to serve as a future example and precedent on how major version bumps should be done.

**Comment by davecgh** [comment]
Needs to be rebased to the latest master.  There are conflicts.

**Pull request merged**